tf_yarn/pytorch/tasks/worker.py

```diff
@@ -14,6 +14,10 @@
 from torch.nn.parallel import DistributedDataParallel as DDP
 from torch.utils.data.distributed import DistributedSampler
 from torch.utils.tensorboard import SummaryWriter
+try:
+    import tensorflow_io as tfio
+except Exception:
+    pass
```

Comment:
why do we need this ?

Reply:
Starting TF2.0, the IO package is separated from TF, and must be imported (e.g. to access hdfs) even if it is not explicitly used. the try block is too keep compatibility with TF<2 as this package is not needed